hr_holiday_notify_employee_manager/models/hr_holidays.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, models
+
+
+class HrHolidays(models.Model):
+    _inherit = 'hr.holidays'
+
+    def _get_approvers_to_notify(self):
+        """Defines who to notify."""
+        company = self.env['res.company']._company_default_get('hr.holidays')
+        if company.leave_notify_manager and self.employee_id.parent_id:
```

Comment:
I did not know that. Yes it does cover my case. Let's just hope that each department as at least one follower.